src/common/decorators/public.decorator.ts

```diff
@@ -0,0 +1,3 @@
+import { SetMetadata } from '@nestjs/common';
+
+export const Public = () => SetMetadata('isPublic', true);
```

Comment:
Add constant

Reply:
?????????????????

Reply:
According to the NestJS documentation you should export the string `isPublic` to a constant for usage in other classes.

Reply:
Okeeeeeeeeeeee gedaan